components/app/explore/similar-datasets/similar-datasets-actions.js

```diff
@@ -0,0 +1,27 @@
+import { createAction, createThunkAction } from 'redux-actions';
+
+// Services
+import DatasetService from 'services/DatasetService';
+
+export const getSimilarDatasetsSuccess = createAction('similar-datasets/getSimilarDatasetsSuccess');
+export const getSimilarDatasetsLoading = createAction('similar-datasets/getSimilarDatasetsLoading');
+export const getSimilarDatasetsError = createAction('similar-datasets/getSimilarDatasetsError');
+export const setSimilarDatasets = createAction('similar-datasets/getSimilarDatasets');
+
+// Async actions
+export const getSimilarDatasets = createThunkAction('similar-datasets/getSimilarDatasets', (datasetId, locale = 'en') => (dispatch) => {
+  dispatch(getSimilarDatasetsLoading());
+  const service = new DatasetService(datasetId, { apiURL: process.env.WRI_API_URL, language: 'en' });
+  return service.getSimilarDatasets()
+    .then((data) => {
+      DatasetService.getDatasets(data.map(d => d.dataset), locale, 'widget,metadata,layer,vocabulary')
```

Comment:
Only one more question: Why are we using `service`but after that we use an static method `DatasetService.getDatasets`?